packages/core-utils/src/itinerary.js

```diff
@@ -495,18 +495,14 @@ export function getTransitFare(fareComponent) {
  * For an itinerary, calculates the transit/TNC fares and returns an object with
  * these values, currency info, as well as string formatters.
  * It is assumed that the same currency is used for transit and TNC legs.
+ *
+ * multiple being set to true will change the output behavior:
```

Comment:
Ok for now, but add FIXME for unifying the returned data structure.

Reply:
Have added a detailed FIXME in 7f5f94945d6ceae3eaf1feeaf966f92b92dba660